guide/src/project_layout.md

```diff
@@ -144,3 +144,17 @@ my-project
 └── src
     └── lib.rs
 ```
+
+## Data
+
+You can add wheel data by creating a `<module_name>.data` folder or setting its location as `data` in `[project.metadata.maturin]`.
```

Comment:
I think we should also allow settings it in `[tool.maturin]` section of `pyproject.toml`?

Reply:
yes, thanks for the reminder!